models/Document/Editable/Video.php

```diff
@@ -701,7 +701,7 @@ private function getYoutubeCode()
         }
 
         $code .= '<div id="pimcore_video_' . $this->getName() . '" class="pimcore_editable_video '. ($config['class'] ?? '') .'">
-            <iframe width="' . $width . '" height="' . $height . '" src="https://www.youtube-nocookie.com/embed/' . $seriesPrefix . $youtubeId . $wmode . $additional_params .'" frameborder="0" webkitAllowFullScreen mozallowfullscreen allowFullScreen data-type="pimcore_video_editable"></iframe>
+            <iframe width="' . $width . '" height="' . $height . '" src="https://www.youtube-nocookie.com/embed/' . $seriesPrefix . $youtubeId . $wmode . $additional_params .'" frameborder="0" webkitAllowFullScreen mozallowfullscreen allowfullscreen allow="fullscreen" data-type="pimcore_video_editable"></iframe>
```

Comment:
Is there any reason for keeping the "allowfullscreen" attribute also? Because, as per the docs, "allowfullscreen" is a legacy attribute and it's redefined as allow = "fullscreen". 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-allowfullscreen:~:text=Note%3A%20This%20attribute%20is%20considered%20a%20legacy%20attribute%20and%20redefined%20as%20allow%3D%22fullscreen%22.

Reply:
Better legacy browser compatibility e.g. Edge 12-18 does not understand `allow="fullscreen"`. Better to avoid breaking anything for now. Especially since these methods are all `private` for some reason, so it cannot be overridden by those who may need to. I can make a second PR against 11.x with only the modern attributes perhaps?

Reply:
@aryaantony92 I see that only the YouTube editable has `data-type="pimcore_video_editable"` but not the others. Is this expected or have they become lost somewhere? Should I add them where missing?

Reply:
@NiklasBr Let me check it

Reply:
@NiklasBr The data-type="pimcore_video_editable" was added to fix an issue with the editing of inherited video editable. So, Could you please add to the others too. Thanks